server/utils.go

```diff
@@ -74,9 +72,9 @@ func (p *Plugin) IsBotDMChannel(channel *model.Channel) bool {
 	return true
 }
 
-func (p *Plugin) sleepUpTo(maxDelay time.Duration) {
-	r := rand.New(rand.NewSource(p.now().UnixNano()))
-	delay := time.Duration(r.Int63n(int64(maxDelay) + 1))
+// func (p *Plugin) sleepUpTo(maxDelay time.Duration) {
+// 	r := rand.New(rand.NewSource(p.now().UnixNano()))
+// 	delay := time.Duration(r.Int63n(int64(maxDelay) + 1))
 
-	time.Sleep(delay)
-}
+// 	time.Sleep(delay)
+// }
```

Comment:
Any objections on just removing this code?

Reply:
Nothing, I ran the test, everything looks fine.